docs/tutorials/05_problem_transformers.ipynb

```diff
@@ -0,0 +1,461 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "5bac463c",
+   "metadata": {},
+   "source": [
+    "# Transforming Problems\n",
+    "\n",
+    "This tutorial explains the various problem transformers available in Qiskit Nature.\n",
+    "\n",
+    "> **Note:** Currently, Qiskit Nature only has transformers which work with electronic structure problems."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f90b6ed9",
+   "metadata": {},
+   "source": [
+    "## The `BasisTransformer`\n",
+    "\n",
+    "This transformer allows you to transform an `ElectronicStructureProblem` from one basis into another.\n",
+    "This is useful in certain settings such as:\n",
+    "- when you obtained a problem description or the Hamiltonian coefficients in the AO basis from an external source\n",
+    "- when you explicitly generated the problem in the AO basis to modify it manually before transforming into the MO basis\n",
+    "- etc.\n",
+    "\n",
+    "Since we can achieve the second scenario directly using Qiskit Nature, that is what we will be doing here. To learn more about how to do this, we recommend that you read the tutorial on the [`QCSchema`](08_qcschema.ipynb)."
```

Comment:
For the three code blocks here it might be helpful to note what they are doing. Eg. We first create a problem in the AO basis to demonstrate the basis transformation, where the objective is to have it in the MO basis. We then create a transformer to go from AO to MO. Finally the problem is transformed and the result can be seen to be in the MO basis.